ui/packages/app/src/project_setting/form/ProjectForm.js

```diff
@@ -13,7 +13,10 @@ import { addToast, EuiComboBoxSelect, useMlpApi } from "@caraml-dev/ui-lib";
 import { ProjectFormContext } from "./context";
 import { EmailTextArea } from "./EmailTextArea";
 import { Labels } from "./Labels";
-import { isDNS1123Label } from "../../validation/validation";
+import {
+  isDNS1123Label,
```

Comment:
is this function still required at all?

Reply:
seems like it's still being used for validating the custom labels added by users